### PR TITLE
Use jasmine instead of mocha for remaining tests.

### PR DESCRIPTION
**Pull request by jimmc**

### CODE REVIEW

File: test/run.sh [hunk 1]
```diff
@@ -101,13 +122,18 @@ function main() {
     trap cleanup INT EXIT SIGHUP SIGINT SIGTERM
   fi
 
-  # Unit tests are fast, run them all first
+  # Unit tests are fast, run them all first, and always run them
```
Comment (yebrahim) [inline test/run.sh]: I think they should still respect the flag, if these grow to a large number the output will be noisy, making it hard to focus on another test suite.

File: test/notebook/test_spec.js [hunk 2]
```diff
@@ -15,36 +15,35 @@
...
-var driver = null;
+require('jasmine2-custom-message');
 
 const notebooks_config = JSON.parse(fs.readFileSync('notebook/spec.json'))
 const timeOut = 60000;
 
-test.before(function() {
-  this.timeout(timeOut);
-  driver = new selenium.Builder()
+function createDriver() {
```
Comment (yebrahim) [inline test/notebook/test_spec.js]: Did `beforeAll` not work here for building the driver once?

File: test/run.sh (hunk 1 above)
Comment (jimmc) [inline test/run.sh]: Done.

File: test/notebook/test_spec.js (hunk 2 above)
Comment (jimmc) [inline test/notebook/test_spec.js]: I had trouble with this before, but I tried it again and now it works. So, Done.

**Pull request merged**